assets/scss/components/_auto-complete.scss

```diff
@@ -29,6 +30,9 @@ Markup:
   </label>
 </fieldset>
 
+ExampleData:
```

Comment:
Can we try this with a space? http://stackoverflow.com/questions/14269989/access-array-key-with-space